Formula/riak.rb

```diff
@@ -1,9 +1,9 @@
 class Riak < Formula
   desc "Distributed database"
   homepage "http://basho.com/products/riak-kv/"
-  url "https://s3.amazonaws.com/downloads.basho.com/riak/2.2/2.2.2/osx/10.8/riak-2.2.2-OSX-x86_64.tar.gz"
-  version "2.2.2"
-  sha256 "73fef949afe3864559fd4ead4cb368749259fac2162a9c931bed20d131f5b8cb"
+  url "https://files-source.tiot.jp/riak/kv/2.2/2.2.3/osx/10.8/riak-2.2.3-OSX-x86_64.tar.gz"
```

Comment:
do you have a reference for this change?

Reply:
I have this URL from the basho repo; and from discussion with some former engineers.  I've considered hosting another public mirror myself as well.

https://github.com/basho/riak/issues/922

2.2.3 was the last release ; I don't think there will be more releases or direction > 2.2.3 until after the court stuff is done.  This PR is to simply keep things functional until that happens.